tidy3d/components/monitor.py

```diff
@@ -579,7 +583,18 @@ def local_origin(self) -> Coordinate:
 
 class FieldProjectionAngleMonitor(AbstractFieldProjectionMonitor):
     """:class:`Monitor` that samples electromagnetic near fields in the frequency domain
-    and projects them at given observation angles.
+    and projects them at given observation angles. The ``center`` and ``size`` fields define
+    where the monitor will be placed in order to record near fields, typically on the order
+    of a wavelength away from the structure of interest. The near fields are then projected
```

Comment:
Maybe rather something like `typically very close to the structure of interest`? I'm wondering if this phrasing makes it sounds like you *should* have some spacing, like a wavelength - while in reality you can put the monitor right next to the structure.